scripts/update_teams.py

```diff
@@ -87,18 +70,11 @@ def create_team(org, name, description, repos):
     all_members.update(contributors)
 
     # Get the github repo for this feedstock.
-    repo = conda_forge.get_repo('{}-feedstock'.format(package_name))
 
     # If the team already exists, get hold of it.
     team = teams.get(package_name)
     if not team:
-        team = create_team(conda_forge, package_name.lower(),
-                           'The {} {} contributors!'.format(choice(superlative), package_name),
-                           [repo])
-        teams[package_name] = team
-    # Ensure that we have merge rights on the repo for this team.
-    url = team.url + "/repos/" + repo._identity
-    team._requester.requestJsonAndCheck("PUT", url, input={"permission": "push"})
+        raise RuntimeError("Team {} does not exist in conda-forge organization".format(package_name))
```

Comment:
Instead of a `RuntimeError`, could we `print` the message and `continue`? Just don't want to break the update script because one feedstock is an undetermined state.